TheMovieDb/HomeViewController.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  ViewController.swift
+//  TheMovieDb
+//
+//  Created by Sandra Herrera on 09/12/20.
+//
+
+import UIKit
+
+class HomeViewController: UIViewController, UICollectionViewDelegate {
+    var viewMoreButton = UIButton()
+    var collectionView: UICollectionView!
+    let service = NetworkManager(urlSession: URLSession.shared)
+    var movies: [Movie] = [] {
+        didSet {
+            collectionView.reloadData()
+        }
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        let layout = UICollectionViewFlowLayout()
+        layout.sectionInset = UIEdgeInsets(top: 25, left: 20, bottom: 10, right: 25)
+        layout.itemSize = CGSize(width: 150, height: 200)
+        collectionView = UICollectionView(frame: self.view.frame, collectionViewLayout: layout)
+        collectionView.register(MovieCell.self, forCellWithReuseIdentifier: "cell")
+        collectionView.dataSource = self
+        collectionView.delegate = self
+        view.addSubview(collectionView)
```

Comment:
lets move all this into another function to provide clarity and not overload a method